layer/tracker/local_execution_project_progress_tracker.py

```diff
@@ -141,34 +142,31 @@ def _update_entity(  # noqa: C901
             self._progress.stop_task(task_id)
             self._progress.update(task_id, completed=0, description="error")
 
-    def mark_derived_dataset_saved(self, name: str, *, id_: uuid.UUID) -> None:
+    def _get_url(self, asset_type: AssetType, name: str) -> URL:
```

Comment:
For upcoming PR: move this to a superclass?

Reply:
👍